stripe.go

```diff
@@ -450,9 +457,20 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 
 // ResponseToError converts a stripe response to an Error.
 func (s *BackendImplementation) ResponseToError(res *http.Response, resBody []byte) error {
+
 	var raw rawError
-	if err := s.UnmarshalJSONVerbose(res.StatusCode, resBody, &raw); err != nil {
-		return err
+	if s.Type == ConnectBackend {
```

Comment:
Yeah, I like this new deserialization logic better — feels cleaner. Thanks for changing.